doc/building.dox

```diff
@@ -58,42 +58,45 @@ Various examples are provided by the @ref building-examples "Example repository"
 @subsection building-packages-vcpkg Vcpkg packages on Windows
 
 Magnum is available as a [Vcpkg](https://github.com/Microsoft/vcpkg) package.
-After setting up Vcpkg like described in the README, you can install latest
-stable version of Magnum with all its dependencies like this:
+After setting up Vcpkg as described in their
+[README](https://github.com/Microsoft/vcpkg#quick-start), it is recommended to
```

Comment:
Well, it's technically not exactly the same link ;) 

I think the first link answers the question "What is vcpkg?" and the second "How do I set it up?" because of their respective contexts. I guess when scrolling down on the first link you'll accidentally discover the Quick Start anyway, so I'd be ok with removing the second.

Reply:
I would expect most people will scroll down to the README anyway, it's usually *the* place to check for install instructions.